Traceback (most recent call last):
  File "/org/groups/ceo/wlao/adjoint/python/Main_no_matrix.py", line 131, in <module>
    t = jnp.linspace(t_span[0], t_span[1], n_steps + 1)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/array.py", line 343, in __getitem__
    return lax_numpy._rewriting_take(self, idx)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/numpy/lax_numpy.py", line 3874, in _rewriting_take
    return lax.dynamic_index_in_dim(arr, idx, keepdims=False)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/lax/slicing.py", line 705, in dynamic_index_in_dim
    result = dynamic_slice_in_dim(operand, index, 1, axis)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/lax/slicing.py", line 698, in dynamic_slice_in_dim
    return dynamic_slice(operand, start_indices, slice_sizes)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/lax/slicing.py", line 110, in dynamic_slice
    return dynamic_slice_p.bind(operand, *start_indices, *dynamic_sizes,
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/core.py", line 360, in bind
    return self.bind_with_trace(find_top_trace(args), args, params)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/core.py", line 363, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/core.py", line 817, in process_primitive
    return primitive.impl(*tracers, **params)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/dispatch.py", line 117, in apply_primitive
    compiled_fun = xla_primitive_callable(prim, *unsafe_map(arg_spec, args),
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/util.py", line 253, in wrapper
    return cached(config._trace_context(), *args, **kwargs)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/util.py", line 246, in cached
    return f(*args, **kwargs)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/dispatch.py", line 208, in xla_primitive_callable
    compiled = _xla_callable_uncached(lu.wrap_init(prim_fun), prim.name,
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/dispatch.py", line 254, in _xla_callable_uncached
    return computation.compile(_allow_propagation_to_outputs=allow_prop).unsafe_call
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 2816, in compile
    self._executable = UnloadedMeshExecutable.from_hlo(
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 3028, in from_hlo
    xla_executable = dispatch.compile_or_get_cached(
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/dispatch.py", line 526, in compile_or_get_cached
    return backend_compile(backend, serialized_computation, compile_options,
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/profiler.py", line 314, in wrapper
    return func(*args, **kwargs)
  File "/org/groups/ceo/wlao/adj-env/lib/python3.10/site-packages/jax/_src/dispatch.py", line 471, in backend_compile
    return backend.compile(built_c, compile_options=options)
jaxlib.xla_extension.XlaRuntimeError: FAILED_PRECONDITION: DNN library initialization failed. Look at the errors above for more details.